### DISCUSSION TITLE
different yarn3 project yarn versions reported for different users?

### DISCUSSION THREAD

**Discussion by pgnd** [yarnpkg/berry#3608] [Q&A] [answered]
I'm migrating a project from Yarn1 -> Yarn3

I've set it up to use 'berry'; as my usual home user, `dev`, the project reports correct yarn ver,
```
whoami
	dev
which yarn
	/var/lib/wwwrun/local/bin/yarn
cd ~
yarn --version
	1.22.17
cd /work/testproj
yarn --version
	3.0.2
```

but, if I switch to my web-dev user, which I'd like to use so that all `yarn` actions create/add/mod files as `UID:GID=wwwrun:www`,
```
su - wwwrun
whoami
	wwwrun
which yarn
	~/local/bin/yarn
ls -ald ~
	drwxr-xr-x 14 wwwrun www 4096 Oct 20 09:42 /var/lib/wwwrun
cd ~
yarn --version
	1.22.17
cd /work/testproj
yarn --version
	1.22.17
```

I've missed/misconfigured something :-/

What needs to be set for my (any) 'other user' to correctly see/use/report the project's set `berry` version?

**Comment by merceyz** [comment, marked as answer]
Make sure your `wwwrun` user doesn't have a `yarnPath` config in `~/.yarnrc(.yml)`

**Comment by pgnd** [comment]
indeed!

```
whoami
	wwwrun
cat ~/.yarnrc
	# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
	# yarn lockfile v1

	lastUpdateCheck 1634729671777
	yarn-path "/var/lib/wwwrun/.yarn/releases/yarn-1.22.17.cjs"

rm -f ~/.yarnrc*
cd /work
yarn --version
	1.22.17
cd /work/testproj
yarn --version
	3.0.2
```

that was quick! thx o/
